### PR TITLE
PP-13234: Adding a high priority feature to test our publish and update workflow

### PR DESCRIPTION
**Pull request by nataliecarey**
In the original README I was pretty clear that I'm not a fan of treating query strings as case insensitive - in our production apps there's no way that `?example=something` will be equivalent to `?EXAMPLE=SOMETHING` but the mock will treat them as equivalent.

The focus of this change is to test the release pipeline, but it also establishes our approach for making the updates we want to make to break strict equivalency with Mountebank.

Note: I used `--save-exact` for the projects using `run-amock` so this won't be automatically installed into those projects, that means that adopting this change will require a pull request which will highlight any tests that need updating.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,21 +73,25 @@ Right now this can be used in the short term as a drop-in replacement for Mounte
...
 
-There are a few things I've learned while going through this exercise, they don't all need changing but I think it's
-worth discussing them in more detail.  Here's a little list as a starter:
 
-### Case insensitive query strings
+### Case sensitive query strings
 
-Mountebank uses case insensitive query strings, both for keys and values.  This will hide problems. At the time of
-writing we have one test that 'uses' the 'feature' (you might be able to tell I'm not a fan).
+Mountebank uses case insensitive query strings, both for keys and values.  This will hide problems. We've
```
Comment (nlsteers) [inline README.md:82]: Is there a bit missing here?

File: package.json [hunk 2]
```diff
@@ -3,12 +3,17 @@
...
     "run-amock": "bin/run"
   },
-  "keywords": ["http", "mock", "govuk", "GOV.UK"],
+  "keywords": [
+    "http",
+    "mock",
+    "govuk",
+    "GOV.UK"
+  ],
   "author": "Natalie Carey <natalie.carey@digital.cabinet-office.gov.uk>",
```
Comment (nlsteers) [inline package.json:17]: I know you've written most of the code in this repo but I think this should be `GOV.UK Pay`, similar to other NPM libs we maintain (which are also majority one person).

File: README.md (hunk 1 above)
Comment (nataliecarey) [reply README.md]: Great spot.  I've added it in.

File: package.json (hunk 2 above)
Comment (nataliecarey) [reply package.json]: Thanks, I've removed it.  I wasn't sure what our standard was.

**Review by nlsteers — APPROVED** [review]

**Pull request merged**